### PR TITLE
More deprecations

### PR DESCRIPTION
**Pull request by minad**
Some minor cleanups in the public API

### CODE REVIEW

File: tommath_class.h [hunk 1]
```diff
@@ -468,7 +464,6 @@
 #endif
 
 #if defined(BN_MP_INIT_MULTI_C)
-#   define BN_MP_ERR_C
```
Comment (minad) [inline tommath_class.h]: @sjaeckel There were many false positives in the dep.pl script like this one for example. Hopefully my fix removed all of them.

Comment (minad) [reply tommath_class.h]: The problem is also that comments are parsed. I should strip them from the source before applying the dep regex.

Comment (sjaeckel) [reply tommath_class.h]: looks very good!

Comment (minad) [reply tommath_class.h]: and fixed

**Pull request merged**